LandSandBoat/server
# Update composure effect to be more retail accurate

**_I affirm:_**

## What does this pull request do?

Updates the scripts/effects/composure.lua to calculate retail accurate accuracy bonus, then applies and removes when appropriate.

## Steps to test these changes

Change job to RDM50 or greater
Use !getstats on yourself and note accuracy bonus
Use job ability composure
Use !getstats on yourself and note increased accuracy bonus
Remove composure effect
Use !getstats on yourself and note accuracy bonus

Your bonus accuracy should be roughly 1/2 of RDM Main level (35 at 70, 45 at 90, etc.)
